tests/test_raft_paper.rs

```diff
@@ -815,7 +815,7 @@ fn test_vote_request() {
         r.step(m).expect("");
         r.read_messages();
 
-        for _ in 0..r.get_election_timeout() * 2 {
+        for _ in 1..r.get_election_timeout() * 2 {
```

Comment:
@xiang90 

We port the newest randomized election timeout changes, but find the our vote request test (port from https://github.com/coreos/etcd/blob/master/raft/raft_paper_test.go#L773) may fail sometimes.